plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/utils/PlayAssetDeliveryUtilsTest.java

```diff
@@ -287,4 +288,27 @@ public void convertDictionaryToAssetPackLocations_typeMismatch2() {
     Map<String, AssetPackLocation> testAssetPackLocations =
         PlayAssetDeliveryUtils.convertDictionaryToAssetPackLocations(testDictionary);
   }
+
+  @Test
+  public void convertExceptionToDictionary_regularException() {
+    Exception testException = new Exception("Just testing, don't panic.");
+    Dictionary testDict = PlayAssetDeliveryUtils.convertExceptionToDictionary(testException);
+
+    Dictionary expectedDict = new Dictionary();
+    expectedDict.put("toString", "java.lang.Exception: Just testing, don't panic.");
```

Comment:
Rather than having a `"toString"` field here, you could use two fields, one for type and one for message.